README.md

```diff
@@ -1,8 +1,65 @@
-# Microverse first repo
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-My first repo of microverse
+# Hello Microverse
 
-## Author
-Michael Hirpo
-mikaelhirpo@gmail.com
+> Description the project.
 
+
+## Built With
+
+- Html
+- css
+
+
+
+
+## Getting Started
+
+**git clone https://github.com/mikihirpo/microverse-first.git**
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/mikihirpo)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/michael-hirpo-40269882/)
+
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] It's great that you have added a link to your MIT.md file which is your license file for this repository. But the problem is MIT.md file doesn't exist in your repo. :smile: Don't worry :relaxed:
- Create a MIT.md file in your main directory of this project.
- Copy the content from this [file](https://github.com/HAMMAS-SALEEM/Hello-Microverse/blob/master/MIT.md) and paste it into your MIT.md file.
- Last but not the least, add your name in place of Hammas Bin Saleem 

That's all :+1: